mlflow/utils/conda.py

```diff
@@ -84,12 +85,54 @@ def _get_conda_executable_for_create_env():
     return conda_env_create_path
 
 
-def _list_conda_environments():
-    prc = process._exec_cmd([get_conda_bin_executable("conda"), "env", "list", "--json"])
-    return list(map(os.path.basename, json.loads(prc.stdout).get("envs", [])))
+def _list_conda_environments(conda_extra_envs=None, return_full_path=False):
```

Comment:
Can we add a docstring for this to explain the input parameters and their behaviors?

Reply:
Please see https://github.com/mlflow/mlflow/pull/5561/files#r844817217 first to determine whether the changes to this method are necessary.